### ISSUE TITLE
Mount/Dismount?

### ISSUE THREAD

**Issue by DavidRieman** [mfriik/dnd5e-quickref#16]
I think the Movement section could use something like a "Mount or Dismount" entry, with "Cost: half movement speed" as the quick tip, and the popup citing, say, PHB 198)

**Comment by mfriik** [comment]
Good Idea just added this as it was a fairly quick thing. If you have any other suggestiond raise another issue :)
